com.unity.shadergraph/Editor/GraphDeltaRegistry/Generation/Interpreter.cs

```diff
@@ -98,12 +101,12 @@ internal static Block EvaluateGraphAndPopulateDescriptors(INodeReader rootNode,
                         if (connectedPort != null) // connected input port-
                         {
                             var connectedNode = connectedPort.GetNode();
-                            mainBodyFunctionBuilder.AddLine($"output.{outputVariables[varIndex++].ReferenceName} = SYNTAX_{connectedNode.GetName()}_{connectedPort.GetName()}");
+                            mainBodyFunctionBuilder.AddLine($"output.{outputVariables[varIndex++].ReferenceName} = SYNTAX_{connectedNode.GetName()}_{connectedPort.GetName()};");
```

Comment:
this is a bug that esme and I fixed at the end of last year, same for the below change